application/src/main/java/org/togetherjava/tjbot/commands/filesharing/GistFiles.java

```diff
@@ -12,14 +12,3 @@
  */
 record GistFiles(@NotNull @JsonIgnore @JsonAnyGetter Map<String, GistFile> nameToContent) {
```

Comment:
Is this one really working? Super weird haha

Reply:
yea I tested it lmao

Reply:
let me test

Reply:
it worked